crates/test/Cargo.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+name = "test"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
```

Comment:
remove this comment

Reply:
Following from this and learning from [rust_analyzer's](https://github.com/rust-lang/rust-analyzer) structure. I'm going to change the internal crates' manifests to version `0.0.0` and to use the workspace definitions e.g.
```toml
[package]
name = "..."
version = "0.0.0"

authors.workspace = true
edition.workspace = true
license.workspace = true
rust-version.workspace = true

[dependencies]
"..." = { workspace = true, ... }
```